src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs

```diff
@@ -310,6 +311,71 @@ public void AddTransferRequest(string jsonSummary, decimal cost, decimal funding
             Publish(() => new TransferRequestCreatedEvent(transferRequest.Id, Id, DateTime.UtcNow, lastApprovedByParty));
         }
 
+		public void DeleteCohort(Party modifyingParty, UserInfo userInfo)
```

Comment:
Trivial, but we could just call this `Delete` as it's obvious it refers to itself